packages/contracts/deploy/new/40_finalize-managing-dao/30_install-multisig-on-managing-dao.ts

```diff
@@ -6,9 +6,14 @@ import {findEvent} from '../../../utils/event';
 import {checkPermission, getContractAddress} from '../../helpers';
 import {Operation} from '../../../utils/types';
 import {hashHelpers} from '../../../utils/psp';
-import {MultisigSetup__factory, Multisig__factory} from '../../../typechain';
+import {
+  MultisigSetup__factory,
+  Multisig__factory,
+  PluginSetupProcessor__factory,
+} from '../../../typechain';
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
 import {InstallationPreparedEvent} from '../../../typechain/PluginSetupProcessor';
+import {DAO__factory} from '../../../typechain/osx-versions/v1_2_0/contracts/core/dao/DAO.sol';
```

Comment:
Same

Reply:
Fixed in https://github.com/aragon/osx/commit/1ac77f580109dd5c4612e7a70899fae68135606f